### ISSUE TITLE
Question: Anyway to have computed properties in validations?

### ISSUE THREAD

**Issue by davidmaitland** [dockyard/ember-validations#220]
Looking at the docs you pass validations as a JSON object on the controller of the model you wish to validate, this works great but I want to be able to use data from the model on the controller for the exclusion definition. Something like this?

```
export default Ember.ObjectController.extend({
  validations: {
    item: {
      presence: true,
      exclusion: {
        in: function() { this.get('model').mapBy('usedItems').compact(); }
      }
    }
  }
});
```

Anyone know how I can do this / or is it not possible?

**Comment by bcardarella** [comment]
It seems you're more interested in treating the `in` property as a function to be evaluated at runtime as opposed to a constant, correct?

**Comment by davidmaitland** [comment]
Yes, well that's my current line of thought. I may be totally confused..

**Comment by bcardarella** [comment]
@davidmaitland to answer the original question: there is no current way of doing this.

However, we should definitely be doing this. I'll mark this as a feature we need for 2.0
